Rocket.Chat/API/Requests/Command/RunCommandRequest.swift

```diff
@@ -13,7 +13,7 @@ typealias RunCommandResult = APIResult<RunCommandRequest>
 typealias RunCommandSucceeded = (RunCommandResult) -> Void
 
 class RunCommandRequest: APIRequest {
-    let method = "POST"
+let method: HTTPMethod = .post
```

Comment:
> >>>

![](https://media.giphy.com/media/m2S7MwDxhVtYs/giphy.gif)

Reply:
😂 damn copy and paste, sorry. I wonder why hound is not reporting those 🤔